package.json

```diff
@@ -55,7 +55,8 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js"
+    "postinstall": "node ./scripts/postinstall.js",
+    "install": "tabtab install --auto"
```

Comment:
Is there a need to maybe check if this is a global installation or does tabtab take care of that?

I ask because we install serverless as a local devDep for a lot of our services rather than a global one and then we `npm run serverless`. we can't take advantage of the autocomplete then, but what about other issues?

i.e. install this as a devDep then `npm install` and see what happens?